modules.d/40network/net-lib.sh

```diff
@@ -101,7 +101,8 @@ get_netroot_ip() {
 }
 
 ip_is_local() {
-    strstr "$(ip route get "$@" 2> /dev/null)" " via "
+    strstr "$(ip route get "$@" 2> /dev/null)" " via " || return 0
+    return 1
```

Comment:
isn't this just `! strstr ...`?

Reply:
Yes, it's the same negation. (I was just following the style of the functions before it).
Thanks

Reply:
The old test is wrong, but now with this one if the `ip` command fails for any reason, `ip_is_local` returns true. Maybe it's good to save the command output in a variable and check that it's not null before calling `strstr`. E.g., see `wait_for_if_link`, `wait_for_if_up`, `wait_for_route_ok`,...

Reply:
good catch, followed up with a commit